packages/kbn-observable/README.md

```diff
@@ -0,0 +1,198 @@
+# `@kbn/observable`
+
+`@kbn/observable` is an observable library based on the [proposed
+`Observable`][proposal] feature and ideas from [RxJS][rxjs].
+
+Why build our own observable library? The main reason is that we don't want to
+tie our plugin apis heavily to a large dependency, but rather expose something
+that's much simpler and that we have control over ourselves.
+
+In addition, `@kbn/observable` implements `System.observable` which enables
```

Comment:
I think this is supposed to be `Symbol.observable`, right?

Reply:
:+1: 🙈